packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Combination/BasicCombinationGraphs/LineWithMultipleBubbleData.test.jsx

```diff
@@ -3,10 +3,12 @@ import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
 import CombinationGraph from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph';
 import '@cerner/terra-graphs-docs/lib/terra-dev-site/ExampleGraphContainer/ExampleGraphContainer.module.scss';
 import getCombinationGraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault';
-import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineWithMultipleBubbleData';
+import lineDataset from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineDataset1';
+import multipleBubbleDataset1 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/bubbleDataset1';
+import multipleBubbleDataset2 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/bubbleDataset2';
 
 const graphConfig = utils.deepClone(getCombinationGraphConfig('#lineWithMultipleBubbleData'));
-const dataset = utils.deepClone(data);
+const dataset = [utils.deepClone(multipleBubbleDataset1), utils.deepClone(multipleBubbleDataset2), utils.deepClone(lineDataset)];
```

Comment:
nit for easier reading:
```suggestion
const dataset = [
                  utils.deepClone(multipleBubbleDataset1), 
                  utils.deepClone(multipleBubbleDataset2), 
                  utils.deepClone(lineDataset)
                ];
```